pkg/sdk/row_access_policies_gen.go

```diff
@@ -110,3 +111,21 @@ type RowAccessPolicyDescription struct {
 	ReturnType string
 	Body       string
 }
+
+// TODO(SNOW-1596962): Fully support VECTOR data type
+func (d *RowAccessPolicyDescription) Arguments() []map[string]any {
+	// Format in database is `(column <data_type>)`
+	plainSignature := strings.ReplaceAll(d.Signature, "(", "")
+	plainSignature = strings.ReplaceAll(plainSignature, ")", "")
+	signatureParts := strings.Split(plainSignature, ", ")
+	arguments := make([]map[string]any, len(signatureParts))
+
+	for i, e := range signatureParts {
+		parts := strings.Split(e, " ")
+		arguments[i] = map[string]any{
+			"name": strings.Join(parts[:len(parts)-1], " "),
```

Comment:
is there some spoecific use case in which we can get list longer than two elements here? If not, can we assert that it should be 2-elements long?